### DISCUSSION TITLE
OSM's Organised Editing Guidelines

### DISCUSSION THREAD

**Discussion by ivanbranco** [teambtcmap/btcmap-data#1513] [General]
Hi,

I noticed you created a page called [Tagging Instructions](https://github.com/teambtcmap/btcmap-data/wiki/Tagging-Instructions/_history) to organize the mapping in the OpenStreetMap database. The OSM guidelines for organised edits requires the documentation to be added to the OSM Wiki as well: [here](https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines#Documentation_on_the_wiki)
There are other rules as well, like a unique hashtag to be used in the changeset comments ecc.ecc. you can find everything in the documentation. There are guidelines for [imports](https://wiki.openstreetmap.org/wiki/Import/Guidelines) and [automated edits](https://wiki.openstreetmap.org/wiki/Automated_edits) as well.

Could you please align with the guidelines before proceeding with further changes?

**Comment by dadofsambonzuki** [comment]
Hi @ivanbranco,

Apologies, we'll get the organised edits page created ASAP and we'll start to use a hashtag in the changesets.

We do not use (nor recommend) automated editing ourselves, but Galoy (a Bitcoin bank) did this yesterday and we are now working with them to help them better understand the process they need to follow going forward.

Our focus is on getting our community members to manual verify old data, increase data quality and take ongoing stewardship of their local areas from a bitcoin merchant perspective.

I'll comment in more detail on the [main thread](https://community.openstreetmap.org/t/shadowy-supertaggers/5543) shortly.

Cheers,

Nathan
Co-founder
BTCMap.org

**Comment by secondl1ght** [comment]
Hi @ivanbranco, the page has now been created here: https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/BTCMap

Could you have a look please and let us know if it checks all of the boxes you were looking for?

Thanks

**Reply by mnalis** [reply]
It talks about `legacy data set`, yet it does not seem defined. In particular, https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines recommends to truthfully describe:
- any non-standard tools and data sources used, and their usage conditions
- links where the community can access any non-standard tools or data sources used

also, check other parts of guidelines, that seem to be missing (or noncomplete), like:
- if the success or performance of participants will be measured in any way, a description of the metrics used for this
- if participants will receive training material or written instructions, a copy of, or link to, these materials
- plans for a “post-event clean up” to validate edits, especially if the activity introduces new contributors to OpenStreetMap

**Comment by dadofsambonzuki** [comment]
Comments have been provided in the [Shadowy Supertagger](https://community.openstreetmap.org/t/shadowy-supertaggers/5543/42) thread.

Our [organised editing ](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/BTCMap )page will be updated based on your feedback. Thanks.
